perf/istio/setup.sh

```diff
@@ -7,63 +7,78 @@ function download() {
 
   local url="https://storage.googleapis.com/istio-prerelease/daily-build/${release}/istio-${release}-linux.tar.gz"
   local outfile="${DIRNAME}/istio-${release}.tgz"
-    
+
   if [[ ! -f "${outfile}" ]]; then
     wget -O "${outfile}" "${url}"
   fi
 
-  echo "${outfile}" 
+  echo "${outfile}"
 }
 
 function install_istio() {
   local DIRNAME="${1:?"output dir"}"
   local release="${2:?"release"}"
-  
+
   local outfile="$(download ${DIRNAME} ${release})"
 
   if [[ ! -d "${DIRNAME}/${release}" ]];then
       tar -xzf "${outfile}" -C "${DIRNAME}"
       mv "${DIRNAME}/istio-${release}/install/kubernetes/helm" "${DIRNAME}/${release}"
       rm -Rf "${DIRNAME}/istio-${release}"
+      helm dep update --skip-refresh "${DIRNAME}/${release}/istio"
```

Comment:
There is a helm update later as well
1. Probably remove the other one
2. what is the purpose of --skip-refresh ?

Reply:
I get the following error with it.

```bash
$ helm dep update tmp/release-1.1-20181108-09-15/istio
Hang tight while we grab the latest from your chart repositories...
...Unable to get an update from the "local" chart repository (http://127.0.0.1:8879/charts):
        Get http://127.0.0.1:8879/charts/index.yaml: dial tcp 127.0.0.1:8879: connect: connection refused
...Successfully got an update from the "stable" chart repository
Update Complete. ⎈Happy Helming!⎈
Saving 14 charts
Deleting outdated charts

```